core/keyserver/keyserver.go

```diff
@@ -116,9 +116,10 @@ func (s *Server) getEntry(ctx context.Context, domainID, userID, appID string, f
 		glog.Errorf("tlog.GetLatestSignedLogRoot(%v): %v", domain.LogID, err)
 		return nil, grpc.Errorf(codes.Internal, "Cannot fetch SignedLogRoot")
 	}
-	// Use the log as the athoritative source of the latest revision.
+	// Use the log as the authoritative source of the latest revision.
 	if revision < 0 {
 		// The maximum index in the log is one minus the number of items in the log.
+		// TODO(phad): could GetTreeSize() return 0?
```

Comment:
It _shouldn't_ because we start the log with at least one item in it.

Reply:
OK, thanks.  If this did occur the math on revision will cause GetLeavesByRevision to fail with an error.